tests/e2e/testutils.go

```diff
@@ -128,6 +128,15 @@ func CreateCluster(nodeOS string, serverCount int, agentCount int) ([]string, []
 	return serverNodeNames, agentNodeNames, nil
 }
 
+func DeleteWorkload(workload, kubeconfig string) error {
+	// Deletes the workload previously applied
```

Comment:
Make this comment a docstring. It might also be nice if this was clear that deletes the content of a manifest file, based on the name it wasn't clear if it deletes a pod or deployment or what.
```suggestion
// DeleteWorkload deletes a previously applied manifest
func DeleteWorkload(workload, kubeconfig string) error {
```